content/en/docs/started/installing-kubeflow.md

```diff
@@ -63,9 +63,9 @@ Packaged distributions are developed and supported by their respective maintaine
         <td>Kubeflow on IBM Cloud</td>
         <td>IBM Cloud</td>
         <td>IBM Cloud Kubernetes Service (IKS) </td>
-        <td>1.3</td>
+        <td>1.4</td>
         <td><a href="/docs/distributions/ibm/">Docs</a></td>
-        <td></td>
+        <td><a href="https://github.com/IBM/manifests/tree/v1.4.0">External Website</a></td>
```

Comment:
@yhwang can you move the changes in this file to another PR? There are different reviewers that owns the started directory.

Reply:
force push and remove change in this file. done